Framework/Core/include/Framework/TableBuilder.h

```diff
@@ -86,6 +87,21 @@ O2_ARROW_STL_CONVERSION(std::string, StringType)
 } // namespace detail
 
 struct BuilderUtils {
+  template <typename T>
+  static arrow::Status appendToList(std::unique_ptr<arrow::FixedSizeListBuilder>& builder, T* data)
+  {
+    using ArrowType = typename detail::ConversionTraits<T>::ArrowType;
+    using BuilderType = typename arrow::TypeTraits<ArrowType>::BuilderType;
+    size_t numElements = static_cast<const arrow::FixedSizeListType*>(builder->type().get())->list_size();
+    
+    auto status = builder->AppendValues(1);
+    auto ValueBuilder = static_cast<BuilderType*>(builder->value_builder());
+    status &= ValueBuilder->AppendValues(data,numElements,nullptr);
+    if (!status.ok()) {
+      throw std::runtime_error("Unable to append values to FixedSizeListBuilder!");
+    }
```

Comment:
Why throwing, rather than simply returning the status?

Reply:
will replace by return status;